packages/docusaurus-mdx-loader/src/remark/rightToc/__tests__/index.test.js

```diff
@@ -10,11 +10,13 @@ import remark from 'remark';
 import mdx from 'remark-mdx';
 import vfile from 'to-vfile';
 import plugin from '../index';
+import slug from '../../slug/index';
 
 const processFixture = async (name, options) => {
   const path = join(__dirname, 'fixtures', `${name}.md`);
   const file = await vfile.read(path);
   const result = await remark()
+    .use(slug)
```

Comment:
Here I use our slugger plugin, because these plugins are interdependent:

https://github.com/facebook/docusaurus/blob/32c1a92b17bdc347add828be97a49991c0611146/packages/docusaurus-mdx-loader/src/index.js#L19

Therefore the tests should take this fact into account.

FYI, rightToc also generated slug, which is actually somewhat redundant, because we already did this in the slugger (ex-remark-slug) plugin.